compiler/rustc_resolve/src/late.rs

```diff
@@ -1151,23 +1180,59 @@ impl<'a: 'ast, 'b, 'ast> LateResolutionVisitor<'a, 'b, 'ast> {
                     .span_label(lifetime.ident.span, note)
                     .emit();
 
+                    self.r.lifetimes_res_map.insert(lifetime.id, LifetimeRes::Error);
+                    return;
+                }
+                LifetimeRibKind::AnonymousPassThrough(node_id) => {
+                    self.r
+                        .lifetimes_res_map
+                        .insert(lifetime.id, LifetimeRes::Anonymous { binder: node_id, elided });
                     return;
                 }
-                LifetimeRibKind::AnonymousCreateParameter
-                | LifetimeRibKind::AnonymousPassThrough
-                | LifetimeRibKind::Item => return,
+                LifetimeRibKind::Item => break,
                 _ => {}
             }
         }
+        self.r
+            .lifetimes_res_map
+            .insert(lifetime.id, LifetimeRes::Anonymous { binder: DUMMY_NODE_ID, elided });
```

Comment:
What does this case mean?
We see an anonymous lifetime, but nothing could introduce it.
Is it some kind of error reported later?